apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -674,6 +674,11 @@ export const platform = {
         { name: 'SSL Enforcement', url: '/guides/platform/ssl-enforcement', items: [] },
       ],
     },
+    {
+      name: 'Billing Management',
```

Comment:
```suggestion
      name: 'Billing',
```

Up to you though, I'd prefer the short version